<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-01-25T22:49:37 IST" name="Suite" finished-at="2024-01-25T22:49:38 IST" duration-ms="422">
    <groups>
    </groups>
    <test started-at="2024-01-25T22:49:37 IST" name="Facebook Login Test" finished-at="2024-01-25T22:49:37 IST" duration-ms="228">
      <class name="TestNGExtent.LoginTest">
        <test-method is-config="true" signature="init()[pri:0, instance:TestNGExtent.LoginTest@5e316c74]" started-at="2024-01-25T22:49:37 IST" name="init" finished-at="2024-01-25T22:49:37 IST" duration-ms="91" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method signature="LoginTest()[pri:0, instance:TestNGExtent.LoginTest@5e316c74]" started-at="2024-01-25T22:49:37 IST" name="LoginTest" finished-at="2024-01-25T22:49:37 IST" duration-ms="6" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Facebook HomePage] but found [Facebook Login]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Facebook HomePage] but found [Facebook Login]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at TestNGExtent.LoginTest.LoginTest(LoginTest.java:43)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method is-config="true" signature="quit()[pri:0, instance:TestNGExtent.LoginTest@5e316c74]" started-at="2024-01-25T22:49:37 IST" name="quit" finished-at="2024-01-25T22:49:37 IST" duration-ms="118" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- TestNGExtent.LoginTest -->
    </test> <!-- Facebook Login Test -->
    <test started-at="2024-01-25T22:49:38 IST" name="Facebook Registration Test" finished-at="2024-01-25T22:49:38 IST" duration-ms="11">
      <class name="TestNGExtent.RegistrationTest">
        <test-method signature="RegisTest()[pri:0, instance:TestNGExtent.RegistrationTest@50a638b5]" started-at="2024-01-25T22:49:38 IST" name="RegisTest" finished-at="2024-01-25T22:49:38 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RegisTest -->
        <test-method is-config="true" signature="init()[pri:0, instance:TestNGExtent.RegistrationTest@50a638b5]" started-at="2024-01-25T22:49:38 IST" name="init" finished-at="2024-01-25T22:49:38 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method is-config="true" signature="quit()[pri:0, instance:TestNGExtent.RegistrationTest@50a638b5]" started-at="2024-01-25T22:49:38 IST" name="quit" finished-at="2024-01-25T22:49:38 IST" duration-ms="8" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- TestNGExtent.RegistrationTest -->
    </test> <!-- Facebook Registration Test -->
  </suite> <!-- Suite -->
</testng-results>
